ui/desktop/CHANGELOG.md

```diff
@@ -2,11 +2,12 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary Desktop.
 
-## Next [CLI 0.7.x] ()
+## v1.4.2 [CLI 0.7.6] (2022.03.16)
 
 ### New
 
 - Support dynamic host catalogs. ([PR](https://github.com/hashicorp/boundary-ui/pull/981))
+- Support SSH type targets. ([PR](https://github.com/hashicorp/boundary-ui/pull/1025))
```

Comment:
Is "Support SSH type targets" misleading here? Currently, we only display that the type is either TCP or SSH. No CRUD operations have been added yet.

Reply:
The desktop client has no CRUD operations, just displays information.

Reply:
Oh right. Thanks for clarifying that. I was thinking big picture, not just the desktop client.